cpp/oneapi/dal/algo/kmeans_init/common.hpp

```diff
@@ -172,6 +177,17 @@ class descriptor : public detail::descriptor_base<Task> {
         base_t::set_seed_impl(value);
         return *this;
     }
+
+    template <typename M = Method, typename = detail::v1::enable_if_plus_plus_dense<M>>
+    auto& get_local_trials_count() const {
```

Comment:
It would be better to provide comments for this user-facing functionality. See `get_seed()` above.

Reply:
Description have been added